app/assets/stylesheets/lists.scss

```diff
@@ -0,0 +1,15 @@
+.list--dash {
+  padding-left: govuk-spacing(3);
+
+  li {
+    position: relative;
+
+    &:before {
+      color: govuk-colour("dark-grey");
+      content: "\2013";
+      left: -16px;
+      position: absolute;
+      top: 0;
+    }
```

Comment:
I have _zero_ knowledge on how we implement CSS correctly within our area, but as I can't easily check this myself due to a lack of domain knowledge, has this been tested on multiple screen sizes with the absolute positioning, e.g. mobile devices?

Reply:
It looks like we're trying to imitate the style of a [detailed guide on GOV.UK](https://www.gov.uk/guidance/content-design/content-types#detailed-guide).

I'd suggest that we use the [Contents list](https://components.publishing.service.gov.uk/component-guide/contents_list) component from the [govuk_publishing_components](https://github.com/alphagov/govuk_publishing_components) gem. That way we'll be using the exact same markup and CSS as is used on the GOV.UK website.

It should be pretty simple to integrate, and then there's less custom CSS for us to maintain and test across browsers & devices, etc. Plus the styles will be consistent with GOV.UK.

Reply:
Ah, I didn't know there's another gem for components

Reply:
I added the styling from the [govuk_publishing_compoenents ](https://github.com/alphagov/govuk_publishing_components). 

We cannot use this gem, it's not compatible with the lates govuk-frontend libarry